### DISCUSSION TITLE
Mapping an organization attribute to user attribute

### DISCUSSION THREAD

**Discussion by bdovaz** [keycloak/keycloak#35497] [Ideas]
Right now there are 2 separate worlds:
* **Group/user level attributes:** based on how you configure it you can flow an attribute from a group to the user having more priority the value of the user attribute if it exists.
* **Organization level attributes:** it is not possible to act as the previous case and these attributes can only be added through the “Organization Membership” mapper but it is a completely separate claim from the concept of group/user attributes and even the format is different.

Until version 25 we only had groups because there was no organization concept until version 26. Now we are evaluating to make the jump because we see that in our case the groups were really treated as organizations.

The problem we have is that the attributes do not flow in the same way, so unless it is possible, we will have to make groups and organizations coexist at the same time without being able to use the attributes at the organization level.

**Comment by ilinaza-vw** [comment]
This is a very interesting feature for us. Could it be considered?

Thank you very much.

**Reply by vramik** [reply]
Created enhancement issue for it, https://github.com/keycloak/keycloak/issues/47381 .. It still needs to be triaged though
